ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -288,9 +291,10 @@ void *compileIRTrace(FN Func, char *FuncNames[], size_t BBs[], size_t TraceLen,
   PassManagerBuilder Builder;
   Builder.OptLevel = 2; // FIXME Make this user-tweakable.
   legacy::FunctionPassManager FPM(JITMod);
+  legacy::PassManager MPM;
   Builder.populateFunctionPassManager(FPM);
```

Comment:
Do we still need the function passmanager or is it superseded by the module passmanager?

Reply:
We need both.

From [the docs](https://llvm.org/doxygen/classllvm_1_1PassManagerBuilder.html#details):
```
A simple example of using it would be:

PassManagerBuilder Builder;
Builder.OptLevel = 2;
Builder.populateFunctionPassManager(FPM);
Builder.populateModulePassManager(MPM);
```